src/BinSkim.Rules/PERules/BA2004.EnableSecureSourceCodeHashing.cs

```diff
@@ -297,5 +302,8 @@ public IEnumerable<IOption> GetOptions()
                 //                RequiredCompilerWarnings,
             }.ToImmutableArray();
         }
+
+        internal static bool IsLikelyUwpDummyObj(Language language, string library, string name) =>
+            language == Language.MASM && library?.Equals(name) == true && library == @"c:\dummy.obj";
```

Comment:
string.Equals call should specify StringComparison, presumably .Ordinal.
Drop "== true"
Don't use == comparison with strings (language and library).  Use .Equals with a comparison specified.

Reply:
fixed.
can not Drop "== true" because there is null check. But I moved it out now so more clear.